bin/cli.js

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env node
+
 const url = require('url')
 const program = require('commander')
 const app = require('../mine.js')
 const pckg = require('../package.json')
 const Web3 = require('web3')
 const ipfsAPI = require('ipfs-api')
+const isIPFS = require('is-ipfs')
```

Comment:
preferred name would be `isIpfs`

Reply:
actually it is used in https://www.npmjs.com/package/is-ipfs examples thats why i used the same